pkg/util/kubeconfig.go

```diff
@@ -33,7 +33,10 @@ func KubeConfigWithClientCerts(
 
 	cluster := clientcmdapi.NewCluster()
 	cluster.Server = clusterURL
-	cluster.CertificateAuthorityData = clusterTrustBundle
+
+	if len(clusterTrustBundle) > 0 {
```

Comment:
Can you explain why this is changed?

Reply:
Previously we always generated kubeconfigs with CertificateAuthorityData, 
now we want to  generate kubeconfigs without it  so we are passing empty `[]byte{}`
 https://github.com/eslutsky/microshift/blob/USHIFT-2458-Certs-RF-Tests/pkg/cmd/init.go#L451